src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -43,14 +42,14 @@
     </PackageReference>
     <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="3.1.0" />
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.6.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.6.1" />
```

Comment:
This brings in the new worker capabilities, allowing use to drastically reduce the grpc payload size for http functions